dmutils/formats.py

```diff
@@ -5,10 +5,12 @@
 
 DATETIME_FORMAT = "%Y-%m-%dT%H:%M:%S.%fZ"
 DATE_FORMAT = "%Y-%m-%d"
+DISPLAY_MONTH_YEAR_FORMAT = "%B %Y"
 DISPLAY_SHORT_DATE_FORMAT = '%-d %B'
 DISPLAY_DATE_FORMAT = '%A %-d %B %Y'
 DISPLAY_TIME_FORMAT = '%H:%M:%S'
 DISPLAY_DATETIME_FORMAT = '%A %-d %B %Y at %I:%M%p %Z'
+DISPLAY_SHORTTIME_LONGDATE_FORMAT = '%I%p %Z, %A %-d %B %Y'
```

Comment:
If you changed this to '%-I%p %Z, %A %-d %B %Y' you could drop the `startswith('0'):` check in `utctoshorttimelongdateformat `, or do you need it available left padded elsewhere?

Reply:
Ah nice, yep.